lib/bolt_server/transport_app.rb

```diff
@@ -280,16 +295,25 @@ def make_winrm_target(target_hash)
       [200, result_set_to_status_hash(result_set, aggregate: aggregate).to_json]
     end
 
+    get '/plans/:module_name/:plan_name' do
+      in_pe_pal_env(params['environment']) do |pal|
+        plan_info = pal.get_plan_info("#{params[:module_name]}::#{params[:plan_name]}")
+        [200, plan_info.to_json]
+      end
+    end
+
     get '/plans' do
```

Comment:
Can we add `@param` docs to these methods to indicate which params are used (and their defaults).